File: app/Article.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;

class Article extends Model
{
    use SoftDeletes;

    protected $table = 'articles';

    protected $dates = ['date', 'deleted_at'];

    protected $fillable = [
        'id',
        'title',
        'date',
        'morgenpost',
        'url',
        'image',
        'excerpt',
        'content'
    ];

    public function author()
    {
        return $this->belongsTo('App\Author');
    }

    public function images()
    {
        return $this->morphMany('App\Image', 'imageable');
    }

    public function crawl()
    {
        return $this->morphOne('App\Crawl', 'crawlable');
    }
}

File: krautreporter/entity/article.go
package entity

import "time"

type Article struct {
	ID        int
	Ordering  int
	Title     string
	Headline  string
	Date      string
	Preview   bool
	URL       string
	Excerpt   string
	Content   string
	CreatedAt time.Time
	UpdatedAt time.Time

	AuthorID int
	Author   *Author
	Images   []Image `gorm:"polymorphic:Imageable;"`
	Crawl    Crawl   `gorm:"polymorphic:Crawlable;"`
}
